src/Compilers/Core/Portable/CodeGen/ReferenceDependencyWalker.cs

```diff
@@ -145,6 +145,25 @@ private static void VisitMethodReference(Cci.IMethodReference methodReference, E
             }
         }
 
+        internal static void VisitSignature(Cci.ISignature signature, EmitContext context)
```

Comment:
>VisitSignature [](start = 29, length = 14)

Do we have a test that makes the work done in this method observable? It would be good to have a NoPia test, for example.